backend/authentication/serializers.py

```diff
@@ -1,10 +1,16 @@
 from django.shortcuts import get_object_or_404
-from rest_framework import serializers
 
-from .models import User, ROLES, ADMIN_ROLES, TheaterAdmin
+from rest_framework import serializers
 
 from theaters.models import Theater
 
+from .models import FanZoneAdmin
+from .models import ROLES
+from .models import TheaterAdmin
```

Comment:
zasto :cry: 
zasto ne u jedan red?

Reply:
Preglednije je, lakse za editovanje i lakse je preimenovati import sa as ako treba. Jeste, uzima vise prostora, slazem se, ali meni to ne znaci. Problem je što nemamo timsku konvenciju i ako budemo ponovo radili nešto u python-u, to će se sigurno desiti.